docs/source/entities-advanced.mdx

```diff
@@ -285,10 +293,161 @@ type Bill @key(fields: "id") {
 
 The [`@override` directive](./federated-types/federated-directives#override) enables you to incrementally migrate between subgraphs with no downtime.
 
-You can migrate between subgraphs gradually with progressive `@override` or all at once.
+Follow these steps to use the `@override` directive:
+
+1. If the `@override` directive isn't already imported, include it in your schema's `@link` imports:
+
+    ```graphql {3} title="Billing subgraph"
+    extend schema
+      @link(url: "https://specs.apollo.dev/federation/v2.7",
+            import: ["@key", "@shareable", "@override"])
+    ```
+
+2. Deploy a new version of the Billing subgraph that both defines and resolves the `Bill` fields you want to move:
+
+    <CodeColumns>
+
+      ```graphql title="Payments subgraph"
+      type Bill @key(fields: "id") {
+        id: ID!
+        amount: Int!
+        payment: Payment
+      }
+
+      type Payment {
+        # ...
+      }
+      ```
+
+      ```graphql {3} title="Billing subgraph"
+      type Bill @key(fields: "id") {
+        id: ID!
+        amount: Int! @override(from: "Payments") #highlight-line
+      }
+      ```
+
+    </CodeColumns>
+
+      Applying the `@override` directive tells the router to resolve the `amount` field in the Billing subgraph instead of the Payments subgraph.
+
+
+3. Update your router's supergraph schema to migrate to the updated Billing subgraph. If you're using managed federation, you do this by publishing the Billing subgraph's schema to GraphOS with [`rover subgraph publish`](/graphos/delivery/#subgraph-schemas).
+
+When the router receives its updated supergraph schema, it immediately starts resolving the `Bill.amount` field from the Billing subgraph while continuing to resolve `Bill.payment` from the Payments subgraph.
+
+<Note>
+
+We can migrate as many entity fields as we want in a single change. To do so, we apply `@override` to every entity field we want to move. We can even migrate entire entities this way.
+
+</Note>
+
+Now that `Bill.amount` is resolved in the Billing subgraph, we can safely _remove_ that field (and its resolver) from the Payments subgraph:
+
+<CodeColumns>
+
+  ```graphql {1-4} title="Payments subgraph"
+  type Bill @key(fields: "id") {
+    id: ID!
+    payment: Payment
+  }
+
+  type Payment {
+    # ...
+  }
+  ```
+
+  ```graphql title="Billing subgraph"
+  type Bill @key(fields: "id") {
+    id: ID!
+    amount: Int! @override(from: "Payments")
+  }
+  ```
+
+</CodeColumns>
+
+After making this change, we deploy our updated Payments subgraph and again update our router's supergraph schema.
+
+<Note>
+
+Because the router is already _ignoring_ `Bill.amount` in the Payments subgraph thanks to `@override`, we can safely publish our updated schema and deploy the subgraph in any order.
```

Comment:
```suggestion
Because the router is already _ignoring_ `Bill.amount` in the Payments subgraph thanks to `@override`, we can safely publish our updated schema or deploy the subgraph in any order.
```